test/pool.js

```diff
@@ -656,6 +665,9 @@ test('pool stream abort in queue', (t) => {
       },
       onComplete () {
         t.pass()
+      },
+      onError (err) {
+        t.strictEqual(err.code, 'UND_ERR_INVALID_ARG')
```

Comment:
This breaks the test. It's testing something different now.

Reply:
Fixed this